.github/workflows/android_tflite_oneshot_build.yml

```diff
@@ -0,0 +1,21 @@
+# Builds the TFLite Java Bindings Android Library using the gradle-android
+# Docker image.
+name: Build TFLite Android Library
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build_android_with_docker:
+    runs-on: ubuntu-latest
+    container: gcr.io/iree-oss/gradle-android@sha256:117aef7fc63facb92003cef6e5869c9759bce44ca51e5152ebc68cca253920ee
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
For some reason, there is a bug in Checkout v2 using the `submodules` command is not compatible with github actions running within docker images (it does however work outside of docker just fine). I'm not certain the reason for this, but googling suggested compatibility issues with v2 and the universal recommendation was to fallback to v1 (which works without issue).

Here is a run showing the issue: https://github.com/KoolJBlack/iree/runs/3784183701?check_suite_focus=true

Reply:
We have other actions that use docker and checkout v2.

The error message in the run you linked also seems pretty clear here:
```
Error: Input 'submodules' not supported when falling back to download using the GitHub REST API. To create a local Git repository instead, add Git 2.18 or higher to the PATH.
```
(update git in the container?)

build_package.yml runs docker like this, rather than using the native `container: ` parameter that you have: https://github.com/google/iree/blob/53f2061277d8d7b0f18be4f1c3c5c122dcff1d27/.github/workflows/build_package.yml#L149-L150

above it uses the checkout v2 action:
https://github.com/google/iree/blob/53f2061277d8d7b0f18be4f1c3c5c122dcff1d27/.github/workflows/build_package.yml#L92-L96

_just_ running the build under docker instead of the entire action seems more stable in general